.github/workflows/Format.yml

```diff
@@ -24,11 +24,32 @@ jobs:
 
       - uses: ./.github/actions/setup-java
 
-      - name: Check spotless
```

Comment:
This PR will remove the KtLint, so we can't autoformat anymore, right?

Reply:
There is a "autoCorrect". does it work? 👀

Reply:
Yes, it's work.
The `detekt-formatting` library is a wrapper for ktlint, so we can continue to use auto format.

https://detekt.dev/docs/intro